dcpy/utils/postgres.py

```diff
@@ -8,41 +8,131 @@
 from typing import Optional
 
 
-BUILD_ENGINE_RAW = os.environ["BUILD_ENGINE"]
-build_engine = create_engine(BUILD_ENGINE_RAW)
+def generate_engine_uri(server_url: str, database: str, schema: Optional[str] = None):
```

Comment:
nit: should the schema default just be "public"?

Reply:
a default sounds good!

I think I didn't make it a PostgresClient method because functions like `postgres.execute_query_via_shell` and `postgres.execute_file_via_shell` can _only_ use an engine URI, so thought this function would be useful outside of PostgresClient